metadata-db-app/src/cmr/metadata_db/services/concept_service.clj

```diff
@@ -654,11 +660,13 @@
 
 (defmethod force-delete-cascading-events :service
   [context concept-type concept-id revision-id]
-  ;; delete the related service associations
-  (delete-associations context concept-type concept-id revision-id :service-association)
-  ;; Do not queue the concept revision deletion event for now.
-  ; (ingest-events/publish-concept-revision-delete-msg context concept-id revision-id)
-  )
+  (when (latest-revision? context concept-id revision-id)
+    ;; delete the related service associations
+    (delete-associations
+     context concept-type concept-id revision-id :service-association)
+      ;; Do not queue the concept revision deletion event for now.
+      (ingest-events/publish-concept-revision-delete-msg
```

Comment:
We are missing tests on search that Elasticsearch is updated correctly from force-deletes. We should have system integration tests for those scenarios (the service is no longer found by search, the collection no longer has the service association).

Reply:
I think Duncan mentioned that he will add the search test later.